web/src/components/storage/PartitionsField.jsx

```diff
@@ -494,18 +461,21 @@ const VolumesTable = ({ volumes, devices, target, targetDevice, isLoading, onVol
     actions: _("Actions")
   };
 
+  /** @type {(volume: Volume) => void} */
   const editVolume = (volume) => {
     const index = volumes.findIndex(v => v.mountPath === volume.mountPath);
     const newVolumes = [...volumes];
     newVolumes[index] = volume;
     onVolumesChange(newVolumes);
   };
 
+  /** @type {(volume: Volume) => void} */
   const deleteVolume = (volume) => {
     const newVolumes = volumes.filter(v => v.mountPath !== volume.mountPath);
     onVolumesChange(newVolumes);
   };
 
+  /** @type {() => React.ReactElement[]|React.ReactElement} */
```

Comment:
Wondering if [`ReactNode`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/67ed83d31b2c8c0381b964b0897fc2457b874d11/types/react/index.d.ts#L448-L489) fits better here, 

```suggestion
  /** @type {() => React.ReactNode} */
```